CodeReview/src/CodeReviewManager.cpp

```diff
@@ -23,3 +23,27 @@
  ** OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  **
  **********************************************************************************************************************/
+
+#include "CodeReviewManager.h"
+
+namespace CodeReview {
+
+// TODO use versions to have a one code review manager for any combination of two versions
+CodeReviewManager::CodeReviewManager(QString, QString)
+{}
+
+CodeReviewManager& CodeReviewManager::instance()
+{
+	// TODO add handling to create instances per version combination
```

Comment:
You can leave it for now, but if you do change it to return multiple instances, definitely use a different name.